examples/cross-mingw/shell.nix

```diff
@@ -0,0 +1,17 @@
+# See docs/cross_compilation.md for details.
+(import <nixpkgs> {
+  crossSystem = {
+    config = "x86_64-w64-mingw32";
+  };
+  overlays = [ (import ../..) ];
+}).callPackage (
+{ mkShell, stdenv, rust-bin, windows }:
+mkShell {
+  nativeBuildInputs = [
+    rust-bin.stable.latest.minimal
+  ];
+
+  buildInputs = [ windows.pthreads ];
+
+  CARGO_TARGET_X86_64_PC_WINDOWS_GNU_LINKER = "${stdenv.cc.targetPrefix}cc";
```

Comment:
Maybe add `wine64` as the test runner? You can check aarch64 example for how to specify it.
https://github.com/oxalica/rust-overlay/blob/db229315e3d220ce90eec3db8fefc944a007be74/examples/cross-aarch64/shell.nix#L13-L18

Reply:
Seemed to work ok. Wine is a bit chatty and has some side effects when running it for the first time but if we are ok with that this should be good to go.